recipes/makeheaders/gh_1.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/.gitignore b/.gitignore
+index 2722579..9c5e82e 100644
+--- a/.gitignore
++++ b/.gitignore
+@@ -1,2 +1,3 @@
+ /makeheaders
+ /makeheaders.c~
++build
+\ No newline at end of file
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..492bad2
+--- /dev/null
++++ b/CMakeLists.txt
```

Comment:
Since this patch was merged upstream (https://github.com/rowanthorpe/makeheaders/commit/5d50a25e12f771a8849eb7e189e4875387bf0dcf), probably drop it and just package a later commit.